### DISCUSSION TITLE
Add Philippines To GitHub Sponsors

### DISCUSSION THREAD

**Discussion by jfullstackdev** [community/community#8712] [Other Feature Feedback, Questions, & Ideas]
Good day. 

Just want to follow up to add the Philippines to your supported countries. Thanks. 

Maybe, I'm the only one requesting it from the Philippines haha. But I will still wait. Thanks.

**Comment by seanmrnda** [comment]
> Good day. 
> 
> Just want to follow up to add the Philippines to your supported countries. Thanks. 
> 
> Maybe, I'm the only one requesting it from the Philippines haha. But I will still wait. Thanks.

Hopefully, few months from now ph will be supported.

**Comment by melvincwng** [comment]
@jdevstatic, great suggestion!

Let's hope Philippines gets added into GitHub Sponsors in the near future 😄 

Would love to see more countries in Asia get added to the list as well ;)

**Reply by japalekhin** [reply]
Hello @melvincwng I can see in your profile that the Sponsor button is active. How did you setup your Stripe account?

**Comment by japalekhin** [comment]
> Good day.
> 
> Just want to follow up to add the Philippines to your supported countries. Thanks.
> 
> Maybe, I'm the only one requesting it from the Philippines haha. But I will still wait. Thanks.

I requeste a long time ago and today, it paid off (kind of). I just got an email saying I'm accepted into the program but I can't set up payments as the only option I see is Stripe Connect. Do you know how to work around this?

**Reply by cevhyruz** [reply]
@japalekhin same. and PH is not yet available on Stripe Connect.

**Reply by jfullstackdev** [reply]
maybe they are just preparing that, not very sure, because when I checked the update in the list of countries, it's not yet included

![image](https://user-images.githubusercontent.com/47092464/181501595-522eb4b2-bba6-422d-a9c9-3478953cc42f.png)

**Reply by cevhyruz** [reply]
@jdevstatic got an email from Stripe 
> We’re currently in a limited pilot in Philippines. We will get back to you as soon as we are ready to onboard more businesses.

I hope they won't take long.

**Reply by jfullstackdev** [reply]
yup hopefully too, any Filipino software developer working in the Philippines here ? just curious

**Reply by japalekhin** [reply]
I'm a Filipino software developer working in the Philippines.

**Reply by jfullstackdev** [reply]
what company ?

**Reply by seanmrnda** [reply]
wish they support paypal and payooner.

**Reply by japalekhin** [reply]
![image](https://user-images.githubusercontent.com/29381739/181581146-1c2084e8-0b0a-4059-b290-04e8b06f2f9d.png)

**Reply by japalekhin** [reply]
Yeah, Paypal would be really good.

**Reply by jfullstackdev** [reply]
> ![image](https://user-images.githubusercontent.com/29381739/181581146-1c2084e8-0b0a-4059-b290-04e8b06f2f9d.png)

oh my what is that error ?

**Comment by queenofcorgis1** [comment]
Yes - Philippines 🇵🇭  is now supported, for more information and other countries now supported, check out the [blog post](https://github.blog/2022-07-28-github-sponsors-available-in-30-new-regions-2/). Thanks for enrolling on our waitlist, submitting this Discussion, and most of all your PATIENCE!

**Comment by japalekhin** [comment]
I just tried setting it up again, Philippines is now on stripe connect options

**Reply by seanmrnda** [reply]
> I just tried setting it up again, Philippines is now on stripe connect options

the problem is, stripe does not support ph.

**Reply by japalekhin** [reply]
> > I just tried setting it up again, Philippines is now on stripe connect options
> 
> the problem is, stripe does not support ph.

Yes they do. But only for Stripe Connect for GitHub Sponsors.

I tried this morning didn't work.

I tried a few mins ago and it worked!

Here are screenshots of my new Stripe Business and Sponsor application:
![Screenshot_2022-07-29-00-52-41-126_com.android.chrome.jpg](https://user-images.githubusercontent.com/29381739/181594555-67d5ab61-9b7e-4f48-bf00-c32645cf0404.jpg)
![Screenshot_2022-07-29-00-51-56-748_com.android.chrome.jpg](https://user-images.githubusercontent.com/29381739/181594579-4de7754a-d47c-4a2e-ab1d-60ff2c2ccab3.jpg)

**Reply by seanmrnda** [reply]
how did you create a stripe account?

**Reply by seanmrnda** [reply]
do you mind telling me the steps that you took?

**Reply by japalekhin** [reply]
I already have a Stripe account. I think anyone from anywhere can sign up for a Stripe account.

The problem is that, you can only enable payments for a business (sub account) for supported countries (no PH).

If you setup via GH Sponsors, it will setup a Stripe Connect business and this supports PH.

**Reply by jfullstackdev** [reply]
yup, that's my idea too even though I'm not yet doing it, 

like when they give swags, all are setup door-to-door delivery

**Comment by japalekhin** [comment]
Update:
![Screenshot_2022-07-29-08-45-03-589_com.google.android.gm.jpg](https://user-images.githubusercontent.com/29381739/181659995-87484535-c00a-4be4-9998-02acb8d75be3.jpg)

**Reply by seanmrnda** [reply]
mine was approved too ;)

**Comment by jfullstackdev** [comment]
someone needs help here about the Stripe account, if you know the answer, please tell us thanks 

https://github.com/orgs/community/discussions/27546
